data/CMakeLists.txt

```diff
@@ -78,6 +78,8 @@ add_custom_command(
         bigram.db
     COMMENT
         "Building binary bigram data..."
+    COMMAND
```

Comment:
I think the import_interpolation tool will modify several files besides bigram.db.